src/components/card/card.ts

```diff
@@ -0,0 +1,48 @@
+import {Component, View, ViewEncapsulation} from 'angular2/core';
+import {CONST_EXPR, Type} from 'angular2/src/facade/lang';
+
+@Component({
+  selector: 'md-card',
+  template: `
+    <div class="md-card">
+      <ng-content></ng-content>
+    </div>
+  `,
+  styleUrls: ['./components/card/card.css'],
+  encapsulation: ViewEncapsulation.None
+})
+export class MdCard {
+}
```

Comment:
Add a comment somewhere in this file mentioning that you use Components without any behavior in order to use content projection to make a cleaner API (by projecting the remainder as the card's "content").